test/data/configClientTest.js

```diff
@@ -0,0 +1,122 @@
+const proxyquire = require('proxyquire')
+
+proxyquire.noCallThru()
+
+describe('configClient', () => {
+  let queryStub
+
+  const mailbox1 = {
+    id: 'id1',
+    establishment: 'establishment1',
+    email: 'email1',
+    name: 'name1',
+    role: 'role1',
+  }
+
+  const mailbox2 = {
+    id: 'id2',
+    establishment: 'establishment2',
+    email: 'email2',
+    name: 'name2',
+    role: 'role2',
+  }
+
```

Comment:
Does this limit the number of emails addresses to 2? I know some staff could potentially have multiple email addresses in NOMIS. If only 2 are needed, its fine.

Reply:
No, just a mock response. Not even used really because we're only testing the inputs, this response is just a dummy. However, looking closer revealed it wouldn't handle an empty result set so I've added a test for that.